### PR TITLE
OC-482 Removed version labels from templates 

### PR DESCRIPTION
**Pull request by tusharraut1994**
Signed-off-by: Tushar Raut tushar@portworx.com

- Removed version labels from a template which was causing issue during statefulset upgrade.
- Updated post-install job to hook

### CODE REVIEW

**Review by sharma-tapas — APPROVED** [review]

File: charts/px-backup/templates/_helpers.tpl [hunk 1]
```diff
@@ -36,9 +36,7 @@ Common labels
 */}}
 {{- define "px-backup.labels" -}}
 app.kubernetes.io/name: {{ template "px-backup.name" . }}
-helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```
Comment (disrani-px) [inline charts/px-backup/templates/_helpers.tpl:39]: This is a required label according to helm, can we keep this? We can remove this from the template labels for the sts

https://helm.sh/docs/chart_best_practices/labels/

Comment (tusharraut1994) [reply charts/px-backup/templates/_helpers.tpl:39]: Added

File: charts/px-backup/templates/_helpers.tpl [hunk 2]
```diff
@@ -36,10 +36,9 @@ Common labels
...
 app.kubernetes.io/instance: {{.Release.Name | quote }}
-app.kubernetes.io/version: {{ .Chart.Version | quote }}
```
Comment (disrani-px) [inline charts/px-backup/templates/_helpers.tpl:41]: Don't need to remove this

File: charts/px-backup/templates/px-central-oidc/pxcentral-keycloak.yaml [hunk 3]
```diff
@@ -231,12 +231,18 @@ spec:
     matchLabels:
       app: postgresql
       role: master
+      app.kubernetes.io/name: {{.Release.Name }}
+      app.kubernetes.io/instance: {{.Release.Name }}
+      app.kubernetes.io/managed-by: {{.Release.Service }}
```
Comment (disrani-px) [inline charts/px-backup/templates/px-central-oidc/pxcentral-keycloak.yaml:236]: Add component label?

File: charts/px-backup/templates/_helpers.tpl (hunk 2 above)
Comment (tusharraut1994) [reply charts/px-backup/templates/_helpers.tpl:41]: Added

File: charts/px-backup/templates/px-central-oidc/pxcentral-keycloak.yaml (hunk 3 above)
Comment (tusharraut1994) [reply charts/px-backup/templates/px-central-oidc/pxcentral-keycloak.yaml:236]: Added

**Pull request merged**